.github/workflows/arc-on-pr.yml

```diff
@@ -0,0 +1,62 @@
+name: TFLint & Plan - ARC
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/arc-*'
+      - 'arc/**'
+      - 'modules/**'
+      - 'scripts/**'
+  push:
+    branches:
+      - main
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  tflint-plan:
+    name: ARC tflint + terraform plan
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+    - name: Install Terraform
+      uses: opentofu/setup-opentofu@592200bd4b9bbf4772ace78f887668b1aee8f716 # v1.0.5
+      with:
+        terraform_version: 1.5.7
+        terraform_wrapper: false
+
+    - uses: terraform-linters/setup-tflint@444635365d380c7363d1eaee4267317c2445717d # v2.0.1
+      name: Setup TFLint
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Install virtualenv
+      run: pip install virtualenv
```

Comment:
What is this for? Also we may need something like
```
echo PATH=${GITHUB_WORKSPACE}/.venv/bin:$PATH >> $GITHUB_ENV
```
to activate the virtual environment for following steps